src/services/scheduler.ts

```diff
@@ -152,6 +152,7 @@ export async function start(dbId: string, dbUser: string) {
       },
       rpc: async (payload: any) => {
         const { params, opid, modulename, methodname } = payload;
+        const rpcConn = await TypeormWrapper.createConn(dbId, { name: uuidv4() });
```

Comment:
hmm but doesn't `upgrade` call `start` at all? making sure that it does seems like a more resource-efficient way of fixing this?

Reply:
no, upgrade just uninstalls and installs

Reply:
perhaps we should have upgrade stop and start the scheduler too? creating a new connection for every call is not what we did on the old path and I am somewhat concerned that it would impose limitations at scale. could we do this another way?

Reply:
I've been trying that start/stop path today, since creating a new connection we need to drop it at the end and that was causing some long-running/infinite runs for some cases... But I'm not fully sure why it is not working, is like it is not getting called...